### ISSUE TITLE
Speaking indicators are distracting

### ISSUE THREAD

**Issue by robintown** [element-hq/element-call#1663]
### Steps to reproduce

1. Join a call
2. Watch other people speak

#### What did you expect?

Speaking indicators should be present to aid in association of audio → participant, but never distracting

#### What happened instead?

Speaking indicators flash on and off too rapidly, and they can become a distraction. I think we really nailed the reactivity back on the full mesh branch, but ever since the switch to LiveKit, they've been feeling a bit too sensitive. Maybe we could contribute to LiveKit upstream to make them a bit less sensitive? If that fails, we could probably hack it in the front-end to make them stick around longer.

### Will you send logs?

No

**Comment by fkwp** [comment]
Maybe we can solve it using https://github.com/livekit/livekit/blob/39d0af43816d14b9266173d9b5a0d66bc2a32bc4/config-sample.yaml#L197 ?

**Comment by Half-Shot** [comment]
I've not personally noticed this, but what do we think now @element-hq/element-call-reviewers ?

**Comment by daniellekirkwood** [comment]
Agree, I think this can be closed as we do this really nicely in product these days. I'm going to close but if this feedback still stands, reopen the ticket! 🙏
